src/eval/tests/mod.rs

```diff
@@ -381,12 +375,12 @@ fn evaluate_num_equal() {
     {
         let expr = "(= 5 5)";
 
-        // TODO: Consider special-casing T, like NIL, and force it to the
+        // TODO: Consider special-casing t, like nil, and force it to the
         // immediate value 1 (with Symbol type-tag). That way boolean logic
         // will work out. It might be more consistent to have an explicit
         // boolean type (like Scheme), though. Otherwise we will have to
         // think about handling of symbol names (if made explicit), since
-        // neither T/NIL as 1/0 will *not* be hashes of their symbol names.
+        // neither t/nil as 1/0 will *not* be hashes of their symbol names.
```

Comment:
This comment can be removed now, rather than updated. This PR collapses the wave function away from any such ideas.